tests/unit_tests/gui/test_csv_export.py

```diff
@@ -72,3 +74,45 @@ def handle_finished_box():
                 f",{name},{gen_kw_data.iloc[i]['COEFFS:a']},{gen_kw_data.iloc[i]['COEFFS:b']},{gen_kw_data.iloc[i]['COEFFS:c']},{misfit_data.iloc[i]['MISFIT:POLY_OBS']},{misfit_data.iloc[i]['MISFIT:TOTAL']}"
                 in file_content
             )
+
+
+@pytest.mark.parametrize("ensemble_select", ["default_0", "*"])
+def test_csv_export(esmda_has_run, qtbot, ensemble_select):
+    gui = esmda_has_run
+
+    file_name = export_data(gui, qtbot, ensemble_select)
+    verify_exported_content(file_name, gui, ensemble_select)
+
+
+def run_experiment_and_export(gui, qtbot):
```

Comment:
Where is the export part? Maybe I just don't know yet how it fully works though :smile:

Reply:
Done, thanks.

Reply:
Ah, good catch. I've renamed to `run_experiment_via_gui`.